ldap-authenticator/src/main/java/org/xwiki/contrib/ldap/internal/LDAPGroupsCache.java

```diff
@@ -101,7 +101,18 @@
      */
     public LRUCacheConfiguration createCacheConfiguration(XWikiLDAPConfig config)
```

Comment:
I would actually remove this method since it does not really make much sense anymore.

Reply:
Or at the very least make deprecate it.

Reply:
I was not sure if this is considered an API, so I left it. Will deprecate it.